ci/jenkins/Jenkins_steps.groovy

```diff
@@ -155,7 +155,7 @@ def compile_unix_int64_gpu() {
         ws('workspace/build-gpu-int64') {
           timeout(time: max_time, unit: 'MINUTES') {
             utils.init_git()
-            utils.docker_run('ubuntu_gpu_cu101', 'build_ubuntu_gpu_large_tensor', false)
+            utils.docker_run('ubuntu_build_cuda', 'build_ubuntu_gpu_large_tensor', false)
```

Comment:
Why this change? Note that `ubunt_build_cuda` is an image that is only provided to workaround some broken design problems

https://github.com/apache/incubator-mxnet/blob/76fa58373636c57fee1e4e6cd7960723b39f455f/ci/docker/Dockerfile.build.ubuntu#L153-L162

We shouldn't migrate more tests to use this image, but rather get rid of this image

Reply:
So the aim is to get rid of both ubuntu_build_cuda and ubuntu_cu101 and instead use `ubuntu_cpu` [base image]
Correct? @leezu

Reply:
Only `ubuntu_build_cuda`, as we still need

https://github.com/apache/incubator-mxnet/blob/76fa58373636c57fee1e4e6cd7960723b39f455f/ci/docker/Dockerfile.build.ubuntu#L143-L150

Reply:
ubuntu_cu101, ubuntu_cu102, etc all make use of that `gpu` target, but have a different base image

Reply:
Wait why not get rid of ubuntu_cu101 as well?

Reply:
We don't want to use libcuda compat right? whereas target of ubuntu_cu101 is gpuwithcompatenv

Reply:
We should get rid of the `gpuwithcompatenv` target and point `ubuntu_cu101` to the `gpu` target.

Reply:
You still need some gpu image. We may later find that (once cuda 11 is released), the gpu target may not be neede anymore and we can point `ubuntu_gpu_cu101` to the `base` target.

`ubuntu_gpu_cu101 would still have a different base image compared to `ubuntu_cpu` `(`nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04` vs `ubuntu:18.04`).

Reply:
Added as recommended
https://github.com/apache/incubator-mxnet/pull/18186/commits/022e135585cd830d2095e6611e13afe6955d21ce
Take a look @leezu

Reply:
Oops didn't do it cleanly. Fixing it now. https://github.com/apache/incubator-mxnet/pull/18186/commits/ec5330d7c86e770687053480be585a75712ce2a4